### PR TITLE
Smal review of doc landing page

### PR DESCRIPTION
**Pull request by tdipisa**
## Description

Some small fixes to the doc entry page.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [X] Other... Please describe: small review of existing entry page doc

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

## Other useful information

[//]: # (rtdbot-start)

URL of RTD document: https://mapstore.readthedocs.io/en/tdipisa-patch-4/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ functionalities (like chart widgets, dashboards, timelines and others).
...
 [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) also provides advanced spatial analysis capabilities that can be used to build WebGIS solutions through a powerful, dynamic and open 
 geospatial application. Since [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is also a framework, you can use it to build your own WebGIS applications by using its plugins and modules.
 
-Last but not the least, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is map agnostic and ensures the greatest flexibility: its abstraction tier allows to [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) to work with different web mapping libraries.
-The supported mapping engines currently supported by [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) are [OpenLayers](https://openlayers.org/) (used by default for desktops), [LeafletJS](https://leafletjs.com/) (used by default for mobile devices) and [Cesium 3D viewer](https://cesiumjs.org/).
+Last but not the least, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is map agnostic and ensures the greatest flexibility: its abstraction tier allows to work with different web mapping libraries.
```
Comment (offtherailz) [inline docs/index.md:17]:
```suggestion
Last but not the least, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) is map agnostic and ensures the greatest flexibility: its abstraction tier allows to work with different web mapping libraries.
```

File: docs/index.md [hunk 2]
```diff
@@ -10,12 +10,12 @@ functionalities (like chart widgets, dashboards, timelines and others).
...
+The mapping engines currently supported by [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) are [OpenLayers](https://openlayers.org/) (used by default for desktops), [LeafletJS](https://leafletjs.com/) (used by default for mobile devices) and [Cesium 3D viewer](https://cesiumjs.org/).
```
Comment (offtherailz) [inline docs/index.md:18]:
```suggestion
The mapping engines currently supported by [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) are [OpenLayers](https://openlayers.org/) (used by default for desktops), [LeafletJS](https://leafletjs.com/) (used by default for mobile devices) and [Cesium 3D viewer](https://cesiumjs.org/).
```

File: docs/index.md [hunk 3]
```diff
@@ -10,12 +10,12 @@ functionalities (like chart widgets, dashboards, timelines and others).
...
 
 [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) has been designed from the beginning to provide a coherent and comprehensive experience across different devices types.
```
Comment (offtherailz) [inline docs/index.md:20]:
```suggestion
[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) has been designed from the beginning to provide a coherent and comprehensive experience across different devices types.
```

File: docs/index.md [hunk 4]
```diff
@@ -10,12 +10,12 @@ functionalities (like chart widgets, dashboards, timelines and others).
 [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) resources are not only related to Maps but also [Dashboards](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-dashboards/) and [Stories](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-stories/); in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) you can create your own innovative and fascinating
```
Comment (offtherailz) [inline docs/index.md:10]:
```suggestion
[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) resources are not only related to Maps but also [Dashboards](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-dashboards/) and [Stories](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-stories/); in [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/ you can create your own innovative and fascinating 
```

**Comment by tdipisa** [comment]
@offtherailz we already have [this issue](https://github.com/geosolutions-it/MapStore2/issues/6587) for what you have mentioned. this PR is aimed to do some small corrections.

**Review by offtherailz — APPROVED** [review]

**Pull request merged**